5-infrastructure/business_unit_1/development/README.md

```diff
@@ -0,0 +1,43 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
We should add a root readme in 5-infrastructure that explains how to deploy this. You can use https://github.com/amandakarina/terraform-example-foundation/tree/feature/adds-step-5/5-app-infra as a starter